appshared/src/test/java/io/reark/rxgithubapp/shared/viewmodels/RepositoriesViewModelTest.java

```diff
@@ -57,52 +60,48 @@
     public void setUp() {
         viewModel = new RepositoriesViewModel(
                 mock(GetGitHubRepositorySearch.class),
-                repositoryId -> Observable.just(mock(GitHubRepository.class)));
+                repositoryId -> Flowable.just(mock(GitHubRepository.class)));
     }
 
     @Test
-    public void testStartFetchingReportedAsLoading() {
-        assertEquals(LOADING, toProgressStatus().call(fetchingStart()));
+    public void testStartFetchingReportedAsLoading() throws Exception {
+            assertEquals(LOADING, toProgressStatus().apply(fetchingStart()));
     }
 
     @Test
-    public void testFetchingErrorReportedAsError() {
-        assertEquals(ERROR, toProgressStatus().call(fetchingError()));
+    public void testFetchingErrorReportedAsError() throws Exception {
+            assertEquals(ERROR, toProgressStatus().apply(fetchingError()));
     }
 
     @Test
-    public void testAnyValueReportedAsIdle() {
+    public void testAnyValueReportedAsIdle() throws Exception {
         GitHubRepositorySearch value = new GitHubRepositorySearch("", Collections.emptyList());
-
-        assertEquals(IDLE, toProgressStatus().call(onNext(value)));
+        assertEquals(IDLE, toProgressStatus().apply(onNext(value)));
     }
 
     @Test
     public void testTooManyRepositoriesAreCappedToFive() {
         TestSubscriber<List<GitHubRepository>> observer = new TestSubscriber<>();
-
-        viewModel.toGitHubRepositoryList()
-                 .call(Arrays.asList(1, 2, 3, 4, 5, 6))
-                 .subscribe(observer);
-
+        try {
+            viewModel.toGitHubRepositoryList()
+                     .apply(Arrays.asList(1, 2, 3, 4, 5, 6))
+                     .subscribe(observer);
+        } catch (Exception e) {
+            Log.e("RepositoriesViewModelTest", e.getMessage());
+        }
         observer.awaitTerminalEvent();
-        assertEquals("Invalid number of repositories",
-                     5,
-                     observer.getOnNextEvents().get(0).size());
+        assertEquals("Invalid number of repositories", 5, observer.getEvents().get(0).size());
     }
 
     @Test
-    public void testTooLittleRepositoriesReturnThoseRepositories() {
+    public void testTooLittleRepositoriesReturnThoseRepositories() throws Exception {
         TestSubscriber<List<GitHubRepository>> observer = new TestSubscriber<>();
-
         viewModel.toGitHubRepositoryList()
-                 .call(Arrays.asList(1, 2, 3))
+                 .apply(Arrays.asList(1, 2, 3))
                  .subscribe(observer);
 
         observer.awaitTerminalEvent();
-        assertEquals("Invalid number of repositories",
-                     3,
-                     observer.getOnNextEvents().get(0).size());
+        assertEquals("Invalid number of repositories", 3, observer.getEvents().get(0).size());
```

Comment:
The reason this fails is that `get(0)` returns one element list within which there is list of 3 elements, use different way to verify the list, the logic in the production code is ok